src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PushCommand.cs

```diff
@@ -92,10 +98,12 @@ public static void Register(CommandLineApplication app, Func<ILogger> getLogger)
                         throw new ArgumentException(Strings.Push_InvalidTimeout);
                     }
 
-                    PackageSourceProvider sourceProvider = new PackageSourceProvider(XPlatUtility.CreateDefaultSettings());
+                    var sourceProvider = new PackageSourceProvider(XPlatUtility.CreateDefaultSettings());
 
                     try
                     {
+                        DefaultCredentialServiceUtility.SetupDefaultCredentialService(getLogger(), !interactive.HasValue());
```

Comment:
nit: formatting looks weird

Reply:
It's fine. I went into VS and reformatted it again and nothing changed.